doc/offboarding.md

```diff
@@ -0,0 +1,16 @@
+# Offboarding
+
+This document is a checklist of things to do when a Collaborator becomes
+Emeritus or leaves the project.
+
+* Remove the Collaborator from the @nodejs/collaborators team.
+* Open a fast-track pull request to move the Collaborator to Collaborator
+  Emeriti in README.md.
+* Email the TSC mailing list to notify TSC members that the Collaborator is
+  moving to Collaborator Emeritus.
+* Determine what GitHub teams the Collaborator belongs to. In consultation with
+  the Collaborator, determine which of those teams they should be removed from.
+    * Some teams may also require a pull request to remove them from a team
```

Comment:
Not sure, but is `them` a bit confusing here? Maybe `a Collaborator` or `someone` etc?